geometry/rotation_body.hpp

```diff
@@ -110,6 +116,64 @@ Rotation<FromFrame, ToFrame>::Rotation(
 }
 
 template<typename FromFrame, typename ToFrame>
+template<typename Scalar, typename F, typename T, typename>
+Rotation<FromFrame, ToFrame>::Rotation(Angle const& angle,
+                                       Bivector<Scalar, FromFrame> const& axis,
+                                       DefinesFrame<ToFrame> tag)
+    : Rotation(AngleAxis(-angle, Normalize(axis).coordinates())) {}
+
+template<typename FromFrame, typename ToFrame>
+template<typename Scalar, typename F, typename T, typename, typename>
+Rotation<FromFrame, ToFrame>::Rotation(Angle const& angle,
+                                       Bivector<Scalar, ToFrame> const& axis,
+                                       DefinesFrame<FromFrame> tag)
+    : Rotation(AngleAxis(angle, Normalize(axis).coordinates())) {}
+
+template<typename FromFrame, typename ToFrame>
+template<typename F, typename T, typename>
+Rotation<FromFrame, ToFrame>::Rotation(
+    Angle const& α,
+    Angle const& β,
+    Angle const& γ,
+    EulerAngles const axes,
+    DefinesFrame<ToFrame> tag)
+    : Rotation(Rotation<ToFrame, FromFrame>(α, β, γ, axes, tag).Inverse()) {}
+
+template<typename FromFrame, typename ToFrame>
+template<typename F, typename T, typename, typename>
+Rotation<FromFrame, ToFrame>::Rotation(
+    Angle const& α,
+    Angle const& β,
+    Angle const& γ,
+    EulerAngles const axes,
+    DefinesFrame<FromFrame> tag)
+    : Rotation(AngleAxis(α, (static_cast<int>(axes) >> (2 * 2)) & 0b11) *
```

Comment:
On second thought, let's not go to LKML, 'tis a silly place.

`BinaryCodedTernaryDigit(k, axes)` sounds reasonable.  Make it constexpr if possible.